src/dev/i18n/utils.js

```diff
@@ -20,21 +20,31 @@
 import {
   isCallExpression,
   isIdentifier,
-  isObjectProperty,
   isMemberExpression,
   isNode,
+  isObjectExpression,
+  isObjectProperty,
+  isStringLiteral,
 } from '@babel/types';
 import fs from 'fs';
 import glob from 'glob';
 import { promisify } from 'util';
 
+import { createFailError } from '../run';
+
 const ESCAPE_LINE_BREAK_REGEX = /(?<!\\)\\\n/g;
 const HTML_LINE_BREAK_REGEX = /[\s]*\n[\s]*/g;
+const VALUES_REFERENCES_REGEX = /{\w+([,\s\w]||({.*}))*}/g;
```

Comment:
I suppose it's a typo here `||`. Only one character `|` is needed for boolean OR.

Reply:
This RegExp matches `{value }`, but does'n match `{ value}` or `{ value }`. I think we should allow spaces after opening curly brace and before closing curly brace or disallow them and mention about it in read.me. What do you think?